_data/devdocs/en/guides/contracts.md

```diff
@@ -210,20 +210,20 @@ all under an Apache license.
 
 {% autocrossref %}
 
-Alice is concerned about her privacy.  She knows every transaction gets
-added to the public block chain, so when Bob and Charlie pay her, they
+Alice is concerned about her privacy. She knows every transaction gets
+added to the public Blockchain, so when Bob and Charlie pay her, they
 can each easily track those satoshis to learn what Bitcoin
 addresses she pays, how much she pays them, and possibly how many
 satoshis she has left.
 
-Alice isn't a criminal, she just wants plausible deniability about
+Alice is not a criminal, she just wants plausible deniability about
```

Comment:
This is an unnecessary change.  The documentation uses contractions quite extensively.